### PR TITLE
Feat:SnackBar | Fix:Avatar Box Shadow

### PR DESCRIPTION
**Pull request by rehanmohiuddin**
- Added Snackbar Simple
- Success,Error Snackbars
- Decreased Box-Shadow Of Code samples (Carbon.sh)

### CODE REVIEW

File: Components/Snackbar/index.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  border-radius: 0.3rem;
+  position: fixed;
+  right: 1rem;
+  top: 5rem;
+  display: none;
+}
+
+.kash-snackbar > div > .kash-btn-text {
+  margin: 0;
+}
```
Comment (tewarimohit) [inline Components/Snackbar/index.css]: no need to be too much spacific. I think only class name will do your thing. Try that..

File: Components/Snackbar/index.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+              style="
+                width: 100%;
+                height: 75vh;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
```
Comment (tewarimohit) [inline Components/Snackbar/index.html]: Can't we define class here? just asking....

**Review by tewarimohit — COMMENTED** [review]
Just go through suggestion once... otherwise code looks good.👍

File: scripts/snackbar.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+["1", "2", "3"].forEach((_id) => {
+  const openBtn_1 = document.getElementById("kash-snackbar-open-btn-" + _id);
+  const _snackbar = document.getElementById("kash-snackbar-" + _id);
+  openBtn_1.onclick = () => {
+    _snackbar.style.display =
+      _snackbar.style.display === "none" ? "flex" : "none";
+  };
+  const closeBtn = document.getElementById("kash-snackbar-close-btn-" + _id);
+  closeBtn.onclick = () => (_snackbar.style.display = "none");
```
Comment (Prabhav-Sharma) [inline scripts/snackbar.js:9]: Why is _snackbar.style.display="none"  in ()?

File: Components/Snackbar/index.css (hunk 1 above)
Comment (rehanmohiuddin) [reply Components/Snackbar/index.css]: Sure

File: Components/Snackbar/index.html (hunk 2 above)
Comment (rehanmohiuddin) [reply Components/Snackbar/index.html]: We can but for every iframe i have kept inline css to maintain uniformity

File: scripts/snackbar.js (hunk 3 above)
Comment (rehanmohiuddin) [reply scripts/snackbar.js:9]: I guess prettier did that previously i wrote one more line which i removed later

**Pull request merged**